src/index.html

```diff
@@ -1,14 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header class="header">
+      <a href="">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__list list">
+          <li class="list__item">
+            <a href="" class="is-active list__link">APPLE</a>
```

Comment:
Не залишай `href` пустим.

Reply:
І текст краще задавати звичайним, а модифікувати його з CSS
```suggestion
            <a href="#" class="is-active list__link">Apple</a>
```